docs/modelling/fulltext.md

```
@@ -0,0 +1,150 @@
+```sql
+SELECT title, _score
+FROM documents
+WHERE MATCH(ft_body, 'search term')
+ORDER BY _score DESC;
+```
+
+**Searching multiple indices with weighted ranking:**
+
+```sql
+MATCH((ft_title boost 2.0, ft_body), 'keyword')
+```
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)